integration_tests/js_handler.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports.hello = async (event) => {
+  return {
+    statusCode: 200,
+    body: JSON.stringify(
+      {
+        message: 'Go Serverless v1.0! Your function executed successfully!',
+        input: event,
+      },
+      null,
+      2
+    ),
+  };
+
+  // Use this code if you don't use the http event with the LAMBDA-PROXY integration
```

Comment:
I'm guessing this file is autogenerated, but we can probably remove this comment